### PR TITLE
MB-1634

### PR DESCRIPTION
**Pull request by crow**
Updates the iOS Xamarin sample app to match the standard iOS sample.

### CODE REVIEW

File: samples/ios-unified/AppDelegate.cs [hunk 1]
```diff
@@ -6,26 +6,132 @@
...
+using System.Security.Authentication.ExtendedProtection;
 namespace Sample
 {
-	[Register ("AppDelegate")]
+	[Register("AppDelegate")]
 	public class AppDelegate : UIApplicationDelegate
 	{
-		public override UIWindow Window {
+		private InboxDelegate inboxDelegate;
+		private PushHandler pushHandler;
```
Comment (madelgi) [inline samples/ios-unified/AppDelegate.cs]: doesn't look like this is used

Comment (crow) [inline samples/ios-unified/AppDelegate.cs]: fixed this

File: samples/ios-unified/PushHandler.cs [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+				return;
+			}
+
+			UIAlertController alertController = UIAlertController.Create(title: notificationContent.AlertTitle,
+																			 message: notificationContent.AlertBody,
+																			 preferredStyle: UIAlertControllerStyle.Alert);
+
+			UIAlertAction okAction = UIAlertAction.Create(title: "OK", style: UIAlertActionStyle.Default, handler: (UIAlertAction action) =>
+			{
+
```
Comment (crow) [inline samples/ios-unified/PushHandler.cs]: fixing this

File: samples/ios-unified/AppDelegate.cs [hunk 3]
```diff
@@ -6,26 +6,147 @@
...
+				else
+				{
+					messageCenterTab.BadgeValue = null;
+				}
+			});
+		}
+
+		private void ShowInvalidConfigAlert()
+		{
+			UIAlertController alertController = UIAlertController.Create(title: "Invalid AirshipConfig.plist", message:
```
Comment (rlepinski) [inline samples/ios-unified/AppDelegate.cs]: should have a new line for the message: param

File: samples/ios-unified/PushHandler.cs [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+			// iOS 10 - let foreground presentations options handle it
+			if (NSProcessInfo.ProcessInfo.IsOperatingSystemAtLeastVersion(new NSOperatingSystemVersion(10, 0, 0)))
+			{
+				completionHandler();
+				return;
+			}
+
+			UIAlertController alertController = UIAlertController.Create(title: notificationContent.AlertTitle,
+																			 message: notificationContent.AlertBody,
```
Comment (rlepinski) [inline samples/ios-unified/PushHandler.cs]: weird formatting here

**Comment by rlepinski** [comment]
👍 after minor nitpicks

**Comment by madelgi** [comment]
:100:

**Pull request merged**